packages/hash/api/src/graph/util.ts

```diff
@@ -160,37 +160,31 @@ export const generateSystemPropertyTypeSchema = (
  */
 export const propertyTypeInitializer = (
   params: PropertyTypeCreatorParams,
-): ((graphApi: GraphApi) => Promise<PropertyTypeWithMetadata>) => {
+): ((context: ImpureGraphContext) => Promise<PropertyTypeWithMetadata>) => {
   let propertyType: PropertyTypeWithMetadata;
 
-  return async (graphApi?: GraphApi) => {
+  return async (ctx?: ImpureGraphContext) => {
     // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition -- @todo improve logic or types to remove this comment
     if (propertyType) {
       return propertyType;
-    } else if (!graphApi) {
+    } else if (!ctx) {
       throw new Error(
-        `property type ${params.title} was uninitialized, and function was called without passing a graphApi object`,
+        `property type ${params.title} was uninitialized, and function was called without passing a impure graph object`,
```

Comment:
Can we change log lines like this in this diff? It's not an impure graph object right? It's a context object that encapsulates the accessors for the API, it's not _a graph_

Reply:
Will call this a `'context object'`

Reply:
9816be5129076f9d6ad32f8369b5e31c103e9f44